### DISCUSSION TITLE
Any way to turn this buttons off?

### DISCUSSION THREAD

**Discussion by griszzza** [qnblackcat/uYouPlus#785] [Q&A]
![image](https://user-images.githubusercontent.com/117314422/208843433-1efb8bf5-3016-456a-a772-343d5743b08e.jpeg)

**Comment by IlyMooon** [comment]
Tell us if you found a way, im still looking for it

**Comment by pha1n0q** [comment]
Tap on the "uYou" tab at the bottom bar of the YouTube app > Tap on the cog icon at the top right > Scroll to and tap on "Video Controls Overlay" > Disable everything except for "Related Videos at the end of videos" and "Cast Button" for the stock YouTube look.
